utils/html.go

```diff
@@ -116,5 +116,32 @@ func (t *HTMLTemplate) RenderToWriter(w http.ResponseWriter) error {
 		l4g.Error(T("api.api.render.error"), t.TemplateName, err)
 		return err
 	}
+
 	return nil
 }
+
+func TranslateAsHtml(t i18n.TranslateFunc, translationID string, args ...interface{}) template.HTML {
```

Comment:
0/5 but does the func sig need to be `args ...interface{}` if we only ever pass it `map[string]interface{}` ?  I might have missed it, but I couldn't find an example of it ever passing anything else.

Reply:
Yeah, I can do that. I was wanting to match the original interface, but that was before I noticed we only ever pass the single map